packages/salesforcedx-vscode-core/src/services/sdr/componentSetUtils.ts

```diff
@@ -52,15 +54,16 @@ export async function setSourceApiVersion(componentSet: ComponentSet): Promise<v
 
   // Next, if it still is not set, set it to the highest API version supported by the target org.
   if (!sourceApiVersion) {
-    const orgApiVersion = await getOrgApiVersion();
-    sourceApiVersion = orgApiVersion ?? componentSet.sourceApiVersion;
+    const orgApiVersion = await exports.getOrgApiVersion();
```

Comment:
When one has:
someFile.ts
```
export function a() {
  b();
}

export function b() {
}
```

When writing unit tests for someFile.ts, one isn't able to mock `b()`.  In order to mock `b()`, one needs to add `'exports.'`, like so:
```
export function a() {
  exports.b();
}

export function b() {
}
```